packages/twenty-server/src/engine/core-modules/ask-ai/ask-ai.service.ts

```diff
@@ -0,0 +1,95 @@
+import { Injectable } from '@nestjs/common';
+
+import { ChatOpenAI } from '@langchain/openai';
+import { SqlDatabase } from 'langchain/sql_db';
+import { PromptTemplate } from '@langchain/core/prompts';
```

Comment:
<img width="471" alt="Screenshot 2024-06-16 at 11 17 45" src="https://github.com/twentyhq/twenty/assets/6399865/6aa035f6-1c34-4ce8-bd32-c574998c3c62">
Quality of response for basic query is still disappointing. I hope we'll manage to get it to something better